Misc/NEWS.d/next/Tests/2021-12-19-12-20-57.bpo-46129.I3MunH.rst

```diff
@@ -0,0 +1,2 @@
+Rewrite ``asyncio.locks`` tests with
```

Comment:
```suggestion
Rewrite :mod:`asyncio.locks` tests with
```

Reply:
`asyncio.locks` is a *private* undocumented module, user code should import everything from `asyncio` top-level package. The module doesn't exist from Sphinx perspective.